### PR TITLE
Implement TextArea text outlining

### PR DESCRIPTION
**Pull request by teep0**

### CODE REVIEW

**Comment by alexeevdv** [comment]
Nice trick. I like it :)

**Comment by isage** [comment]
:+1:

File: src/UI/TextArea.cpp [hunk 1]
```diff
@@ -260,6 +281,39 @@ void TextArea::_calculate()
...
+                outlineSymbolTop->setFont(outlineFont);
+                
+                _symbols.push_back(*outlineSymbolLeft);
+                _symbols.push_back(*outlineSymbolTopLeft);
+                _symbols.push_back(*outlineSymbolBottomLeft);
+                _symbols.push_back(*outlineSymbolRight);
+                _symbols.push_back(*outlineSymbolTopRight);
+                _symbols.push_back(*outlineSymbolBottomRight);
+                _symbols.push_back(*outlineSymbolBottom);
+                _symbols.push_back(*outlineSymbolTop);
```
Comment (phobos2077) [inline src/UI/TextArea.cpp]: Maybe reduce code repetition?

Comment (isage) [inline src/UI/TextArea.cpp]: Any ideas how? Without making this unreadable (e.g. doing this all in a cycle)
Helper inline function or macros, maybe?

Comment (phobos2077) [inline src/UI/TextArea.cpp]: Simply putting repeated stuff in one function would do the job I think, on "client-side" you just tell which offsets you need for each outline. That's DRY principle :)

**Pull request merged**